lib/routes/account.js

```diff
@@ -659,6 +659,21 @@ module.exports = function (
               return true
             }
           }
+
+          // If the account was recently created, don't make the user
+          // confirm sign-in for a configurable amount of time. This will reduce
+          // the friction of a user adding a second device.
+          const bypassAccountWithAge = config.signinConfirmation.bypassAccountWithAge
+          const accountCreatedSince = Date.now() - account.createdAt
```

Comment:
@rfk I was pretty indifferent to the naming but yours are much clearer. `maxAge` vs `accountCreatedSince`, makes way more sense in hindsight. Will update!